deploy.sh

```diff
@@ -41,11 +83,10 @@ else
 fi
 
 if [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
-	"${CONTAINER_CMD:-docker}" login -u "${QUAY_IO_USERNAME}" -p "${QUAY_IO_PASSWORD}" quay.io
```

Comment:
let the docker login be here only and call `build_push_images` here only

Reply:
will change